### ISSUE TITLE
[Machine Translation] Machine translate all relevant files

### ISSUE THREAD

**Issue by roadlittledawn** [newrelic/docs-website#5162]
## Summary

Once all the work is done to set up machine translation and the `feature/machine-translation` branch is merged into `develop`, we want to:
- write a script that we can run once to machine translate all relevant files (and keep it around so we can use subsequently), 
- create a separate branch for just the translated files, 
- and create a PR (to `develop`) and merge it.

### Relevant files
* File paths to exclude:
  * `src/content/docs/release-notes`
  * `src/content/docs/licenses`
  * `src/content/docs/style-guide`
  * `src/content/docs/agile-handbook`
  * `src/content/whats-new`
  * `src/data-dictionary`
* Content types (via frontmatter) to exclude:
  * `apiLandingPage` (these will eventually go away completely)

## Script
<Details>

Normally, a PR will contain a set of files, and that triggers the first workflow to look at that set of files and queue them up for translation. 

Since we want this to run for some subset of files manually, we will need a custom script.

The script will need to:
- identify all non-human translate pages -- pages we want to machine translate
- filter out exclusions mentioned above
- submit records to database directly. This criteria isn't the only way to do this, but is the simplest imo. We can also test this locally just to be sure.

</Details>

## Acceptance criteria
- [ ] Write a script that, given a file directory, can be run to create a PR for given files that we want to machine translate
- [ ] Relevant file paths / content types are added to `i18n-exclusions.yml`
- [ ] Script is run once on all relevant files and results in a PR
- [ ] The PR is merged (build check passes)
- [ ] If there are build failures because of MDX parsing / deserializing issues, create follow up ticket to try to permanently fix

**Comment by clarkmcadoo** [comment]
@roadlittledawn regarding the exclusions we want to add, will this be for all various locales or just one in particular (say ja-JP for now 😄 )

**Comment by jmiraNR** [comment]
Hello, Could these pages be excluded from the Machine translation workflow? 
Data privacy: https://docs.newrelic.com/docs/security/security-privacy/data-privacy/
Security compliance: https://docs.newrelic.com/docs/security/security-privacy/compliance/
Thank you.

**Comment by clarkmcadoo** [comment]
@zstix -- leaving this in the "In Review" column until the PR with all of the Machine Translations have been completed, @nr-kkenney came up with the plan and knows what's going on with that 🙏  👍

**Comment by roadlittledawn** [comment]
update from workflow run this morning:

- we see: 630 files and ~310k words. we queued 1,437 files
- one error is no serializer for html element strong. and many authentication errors which is suspicious. See Smartling doc on error codes 401 unauthorized

**Comment by nr-kkenney** [comment]
going to close this ticket and track the follow up work per clinton's last comment in https://github.com/newrelic/docs-website/issues/5348.

i think this is fair since this update does what we want it to do and does work, its just that we hit an edge case for our overall process -- and not specifically this effort -- that we didn't factor in.
